lib/unhangout-server.js

```diff
@@ -111,6 +111,16 @@ exports.UnhangoutServer.prototype = {
             NODE_ENV: process.env.NODE_ENV
         }
 
+        // Much less repetitive to include these here, both for consistency
+        // and because _header.ejs will only have access to these and
+        // contexts passed in res.render().
+        if (!_.isEmpty(this.options.CUSTOM_CSS_FILES)) {
+            this.app.locals.customCssFiles = this.options.CUSTOM_CSS_FILES;
+        }
+        if (!_.isEmpty(this.options.CUSTOM_FACILITATOR_CSS_FILES)) {
+            this.app.locals.customFacilitatorCssFiles = this.options.CUSTOM_FACILITATOR_CSS_FILES;
+        }
```

Comment:
Because these don't get set to app.locals if they're empty, all the tests are failing, and rendering will fails.  Could you update this to always define ``this.app.locals.customCssFiles`` and ``this.app.locals.customFacilitatorCssFiles``, even if they're empty, so that it doesn't mess up the template renderer?